tla-import/src/test/scala/at/forsyte/apalache/tla/imp/TestSanyImporter.scala

```diff
@@ -1499,6 +1499,27 @@ class TestSanyImporter extends FunSuite {
     assert(aOfM.isInstanceOf[TlaOperDecl] && aOfM.asInstanceOf[TlaOperDecl].formalParams.size == 1)
   }
 
+  test("substitution in INSTANCE") {
```

Comment:
@konnov is this related to the Etc work? Is it because of a regression that was introduced in the etc stuff, or is it a misplaced change?

Reply:
I think it was just a quick fix/test for a type-checking benchmark that caused a failure in the parser.